Libraries/LibWeb/HTML/Navigable.cpp

```diff
@@ -816,7 +816,23 @@ static WebIDL::ExceptionOr<Navigable::NavigationParamsVariant> create_navigation
     request->set_referrer(entry->document_state()->request_referrer());
     request->set_policy_container(source_snapshot_params.source_policy_container);
 
-    // 4. If documentResource is a POST resource, then:
+    // 4.  If request's client is null, then:
```

Comment:
Extra space here, and a couple more below you can find by searching for `.  `

Reply:
artifact from copying steps from the whatpr url it seems. I'll do a scan.